tests/scripts/check_stats.py

```diff
@@ -1738,6 +1747,56 @@ def load_scalar_file(options,selector):
 
 
 
+######################################################################
+#                    IMPORTANT DEVELOPER INFO                        #
+######################################################################
+# Information for individual quantities in stat.h5 file.
+#   For each quantity, list a default label and data paths.
+#   To add new quantities to be checked, update this data structure.
+stat_info = obj({
+    'density'       : obj(
+        default_label = 'Density',
+        data_paths    = obj(tot='value'),
+        ),
+    'spindensity'   : obj(
+        default_label = 'SpinDensity',
+        data_paths    = obj(u='u/value',
+                            d='d/value'),
+        ),
+    'energydensity' : obj(
+        default_label = 'EnergyDensity',
+        data_paths    = obj(u='number_matrix/u/value',
+                            d='number_matrix/d/value'),
+        ),
+    '1rdm'          : obj(
+        default_label = 'DensityMatrices',
+        data_paths    = obj(u='energy_matrix/u/value',
+                            d='energy_matrix/d/value'),
+        ),
+    '1redm'         : obj(
+        default_label = 'DensityMatrices',
+        data_paths    = obj(u='number_matrix/u/value',
+                            d='number_matrix/d/value'),
+        ),
+    'obdm'          : obj(
+        default_label = 'OneBodyDensityMatrices',
+        data_paths    = obj(W=('spacegrid1/value',0,3),
+                            T=('spacegrid1/value',1,3),
+                            V=('spacegrid1/value',2,3)),
+        ),
+    'momentum'      : obj(
+        default_label = 'nofk',
+        data_paths    = obj(tot='value'),
+        ),
+    'sh_coeff'      : obj(
+        default_label = 'sh_coeff',
+        data_paths    = obj(coeff='value'),
+        ),
+    })
```

Comment:
Incorrect mapping in the above caused test failure. Please check against the old table.
```
        quantity_paths = obj({
        qpaths = stat_info[options.quantity].data_paths
            'density'       : obj(tot='value'),
            'spindensity'   : obj(u='u/value',
                                  d='d/value'),
            '1rdm'          : obj(u='number_matrix/u/value',
                                  d='number_matrix/d/value'),
            '1redm'         : obj(u='energy_matrix/u/value',
                                  d='energy_matrix/d/value'),
            'obdm'          : obj(u='number_matrix/u/value',
                                  d='number_matrix/d/value'),
            'energydensity' : obj(W=('spacegrid1/value',0,3),
                                  T=('spacegrid1/value',1,3),
                                  V=('spacegrid1/value',2,3)),
            'momentum'      : obj(tot='value'),
```